packages/forma-36-react-components/src/components/Card/Card.tsx

```diff
@@ -46,57 +46,55 @@ interface CardPropTypes extends BaseCardProps {
    * Any additional styles that are being applied
    */
   style?: React.CSSProperties;
-  children: React.ReactNode;
 }
 
-const defaultProps: Partial<CardPropTypes> = {
+const defaultProps: Partial<CardProps> = {
   padding: 'default',
   testId: 'cf-ui-card',
   selected: false,
 };
 
-export class Card extends Component<CardPropTypes> {
-  static defaultProps = defaultProps;
+export const Card = ({
+  className,
+  testId,
+  children,
+  href,
+  target,
+  onClick,
+  padding,
+  selected,
+  ...otherProps
+}: CardProps) => {
+  const classNames = cn(styles.Card, className, {
+    [styles[`Card--padding-${padding}`]]: padding,
+    [styles['Card--is-interactive']]: onClick || href,
+    [styles['Card--is-selected']]: selected,
+  });
 
-  handleClick = (event: ReactMouseEvent<HTMLElement>) => {
-    if (!this.props.onClick) return;
+  const Element: React.ElementType = href ? 'a' : 'div';
 
-    this.props.onClick(event);
-  };
-
-  render() {
-    const {
-      className,
-      testId,
-      children,
+  return React.createElement(
```

Comment:
should we replace this with the common `return (...)`?

Reply:
I prefer also hahaha
I didn't replace it because I thought there was a reason to use the createElement

I will refactor it! thanks :)